src/components/PasswordInput/PasswordInput.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+    <input
+        :value="value"
```

Comment:
I no sure are you still need the `value` as props.
because I didn't see you pass anything in `value` props